src/pages/SettingsPage/AddonsManager/helpers.js

```diff
@@ -103,8 +119,16 @@ export const transformVersionsTable = (data, addons = [], deletedVersions) => {
       if (status.length === 0 && bundlesArray.length > 0) {
         status.push('active')
       }
+      // check if version is broken
+      if (version.isBroken) {
+        status.push('error')
+        suffix = '(broken)'
+        tooltip = 'Addon failed to load. Check logs for details' //JSON.stringify(version.reason)
```

Comment:
Should this second sentence have ended with a dot `.` as well?

Reply:
Good spot but this will be gone soon and replaced with `version.reason`